### DISCUSSION TITLE
Display name option for GSA website - Forums

### DISCUSSION THREAD

**Discussion by Loops117** [gameserverapp/Platform#854] [Suggestions]
This is mainly for the forum section that comes with my GSA website. I created my account on the website using my google account. Unfortunately, my full real name is being displayed in the forums, rather then a character name.

We need an option to set a display name on the website. This way i can use the name everyone in my community knows me by instead of using my legal full name, which i'd rather people didnt have access to.

**Comment by mvaessen** [comment]
Forum has been removed, so closing this
